concepts/05 UI Components/Lookup/00 Getting Started with Lookup/40 Customize the Drop-Down Menu.md

```diff
@@ -0,0 +1,81 @@
+On desktops and iOS devices, Lookup's drop-down menu is the [Popover](/concepts/05%20UI%20Components/Popover/00%20Overview.md '/Documentation/Guide/UI_Components/Popover/Overview/') UI component; on other devices, it is the [Popup](/concepts/05%20UI%20Components/Popup/00%20Getting%20Started%20with%20Popup '/Documentation/Guide/UI_Components/Popup/Getting_Started_with_Popup/') component.
+
+To customize Popup or Popover, use the [dropDownOptions](/api-reference/10%20UI%20Components/dxLookup/1%20Configuration/dropDownOptions.md '/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#dropDownOptions') object. For example, the following code allows users to close Lookup on outside click and removes the item list title:
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#lookupContainer").dxLookup({
+            // ...
+            dropDownOptions: {
+                closeOnOutsideClick: true,
+                showTitle: false
+            }
+        });
+    });
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-lookup>
+        <dxo-drop-down-options
+            [closeOnOutsideClick]="true"
+            [showTitle]="false">
+        </dxo-drop-down-options>
+    </dx-lookup>
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxLookup>
+            <DxDropDownOptions
+                :close-on-outside-click="true"
+                :show-title="false"
+            />
+        </DxLookup>
+    </template>
+
+    <script>
+    // ...
+
+    import { DxLookup, DxDropDownOptions } from 'devextreme-vue/lookup';
+
+    export default {
+        components: {
+            DxLookup,
+            DxDropDownOptions
+        },
+        data() {
+            return {
+                // ...
+            };
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import { Lookup, DropDownOptions } from 'devextreme-react/lookup';
+
+    function App() {
+        return (
+            <Lookup>
+                <DropDownOptions
+                    closeOnOutsideClick={true}
+                    showTitle={false}
+                />
+            </Lookup>
+        );
+    }
+    export default App;
+
+---
+
+You have configured basic Lookup features. To take a more detailed look at this UI component, explore the following resources:
```

Comment:
- `You have configured basic Lookup features` I don't know why you mention this - if you follow the tutorial, you already know this
- `To take a more detailed look at this UI component` What do es this imply? These resources give you more information? more examples?

Reply:
1) I wrote this as a conclusion to kind of wrap the whole text up. If you think that this sentence is unnecessary, I will delete it.
2) Yes, these resources give both examples and more information.

Reply:
Perhaps: For more information about this UI component and examples, refer to the following resources:

Reply:
Noted